src/state/ducks/account.js

```
@@ -276,11 +272,15 @@ function* submitFacebookCodeSaga({ payload }) {
 }
 
 function* checkWarningOfEmptyAccounts() {
-    const results = yield call(getAccountsSaga);
-    yield put({
-        type: SET_VISIBILITY_OF_EMPTY_ACCOUNTS_WARNING,
-        payload: getNonExistedRequiredAccounts(results).length > 0,
-    });
+    try {
+        const results = yield call(getAccountsSaga);
+        yield put({
+            type: SET_VISIBILITY_OF_EMPTY_ACCOUNTS_WARNING,
+            payload: getNonExistedRequiredAccounts(results).length > 0,
+        });
+    } catch (e) {
```

@lkazberova Why do we catch the error here if we're just going to log it to the console?  It doesn't help the user at all and it has essentially the same effect as not catching it (it gets logged to console).  Does it break the React app?  My primary concern is that we'll never actually see it because it silently fails.  We're using Sentry now to capture uncaught exceptions (https://sentry.io/organizations/primer-labs-inc/issues).  How can we change our code to ensure that we catch these issues?

---

fixed